### DISCUSSION TITLE
Nametagedit

### DISCUSSION THREAD

**Discussion by Blueeyestar** [Zrips/CMI#733] [Feature Requests]
It's a really great plugin.
only two roofs are missing what would make it perfect.
1. a nametagedit function -> / nte player NAME prefix '[admin]'

**Comment by PlopHat** [comment]
Theres a command called /cmi nameplate that changes tag
/cmi nameplate NAME -pref:PREFIX -suf:SUFFIX

**Comment by PlopHat** [comment]
I think that only works with users ans not groups

**Comment by Blueeyestar** [comment]
Unfortunately not to use when the tray recharges everything is gone again

**Comment by x1p** [comment]
There doesn't seem to be a "clear" option
And I don't see anything happen.
But maybe I'm missing something.

**Comment by 0xC1A551C** [comment]
animatednames plugin.

**Comment by MCTyler** [comment]
@0xC1A551C The whole purpose of this 1 plugin is to get rid of all the other smaller ones....JS

**Comment by ACluco1** [comment]
Hey @Zrips, can you add this feature? We really want it :P
